mmselfsup/models/algorithms/mae.py

```diff
@@ -17,7 +22,11 @@ class MAE(BaseModel):
             Defaults to None.
     """
 
-    def __init__(self, backbone=None, neck=None, head=None, init_cfg=None):
+    def __init__(self,
+                 backbone: dict = None,
```

Comment:
```suggestion
                 backbone: Optional[dict] = None,
```
Please also check other parts